docs/design_assets/ReadMe.md

```diff
@@ -0,0 +1,17 @@
+# Electra
+
+<p align="center">
```

Comment:
Excepted some <br>, we should really avoid using HTML tags in MD files. It's accepted but kind of an anti-pattern since the Markdown format has been created in order to allow a nice readability under text-only editors too.

Reply:
You are right. I simply moved from documents repo to this one. I will fix it.